<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>511</width>
    <height>837</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background:white;</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <item>
    <widget class="QWidget" name="widget" native="true">
     <property name="styleSheet">
      <string notr="true">border-radius:10px;</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <property name="bottomMargin">
       <number>9</number>
      </property>
      <item>
       <widget class="QWidget" name="widget_2" native="true">
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>25</height>
         </size>
        </property>
        <widget class="QWidget" name="widget_9" native="true">
         <property name="geometry">
          <rect>
           <x>380</x>
           <y>0</y>
           <width>80</width>
           <height>27</height>
          </rect>
         </property>
         <property name="minimumSize">
          <size>
           <width>80</width>
           <height>0</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>70</width>
           <height>16777215</height>
          </size>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout_4">
          <property name="leftMargin">
           <number>0</number>
          </property>
          <property name="topMargin">
           <number>0</number>
          </property>
          <property name="rightMargin">
           <number>0</number>
          </property>
          <property name="bottomMargin">
           <number>0</number>
          </property>
          <item>
           <widget class="QPushButton" name="pushButton_minimize">
            <property name="minimumSize">
             <size>
              <width>10</width>
              <height>10</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>25</width>
              <height>25</height>
             </size>
            </property>
            <property name="font">
             <font>
              <pointsize>8</pointsize>
              <weight>75</weight>
              <bold>true</bold>
             </font>
            </property>
            <property name="styleSheet">
             <string notr="true">border:none;</string>
            </property>
            <property name="text">
             <string/>
            </property>
            <property name="icon">
             <iconset>
              <normaloff>:/newPrefix/2931142_minimize_minus_remove_line_icon.png</normaloff>:/newPrefix/2931142_minimize_minus_remove_line_icon.png</iconset>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="pushButton_close">
            <property name="maximumSize">
             <size>
              <width>25</width>
              <height>25</height>
             </size>
            </property>
            <property name="styleSheet">
             <string notr="true">border:none;</string>
            </property>
            <property name="text">
             <string/>
            </property>
            <property name="icon">
             <iconset>
              <normaloff>:/newPrefix/close.png</normaloff>:/newPrefix/close.png</iconset>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </widget>
      </item>
      <item>
       <widget class="QWidget" name="widget_3" native="true">
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>55</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">margin-bottom:0px;</string>
        </property>
        <widget class="QPushButton" name="btn_login_1">
         <property name="geometry">
          <rect>
           <x>130</x>
           <y>10</y>
           <width>111</width>
           <height>41</height>
          </rect>
         </property>
         <property name="font">
          <font>
           <family>Arial</family>
           <pointsize>11</pointsize>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">background-color:#60FF42;
border:none;
border-radius:0px;
border-right:1px solid black;
</string>
         </property>
         <property name="text">
          <string>Login</string>
         </property>
        </widget>
        <widget class="QPushButton" name="register_btn_1">
         <property name="geometry">
          <rect>
           <x>240</x>
           <y>10</y>
           <width>121</width>
           <height>41</height>
          </rect>
         </property>
         <property name="font">
          <font>
           <family>Arial</family>
           <pointsize>11</pointsize>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">background-color:#1DB700;
border:none;
border-radius:0px;
border-left:2px solid black;
</string>
         </property>
         <property name="text">
          <string>Register</string>
         </property>
        </widget>
       </widget>
      </item>
      <item>
       <widget class="QWidget" name="widget_4" native="true">
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>700</height>
         </size>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_2">
         <property name="topMargin">
          <number>0</number>
         </property>
         <item>
          <widget class="QTabWidget" name="tabWidget">
           <property name="styleSheet">
            <string notr="true">background:#86FF70;
border:none;
box-shadow: none;</string>
           </property>
           <property name="currentIndex">
            <number>0</number>
           </property>
           <widget class="QWidget" name="tab">
            <attribute name="title">
             <string>Tab 1</string>
            </attribute>
            <widget class="QWidget" name="widget_26" native="true">
             <property name="geometry">
              <rect>
               <x>0</x>
               <y>40</y>
               <width>450</width>
               <height>391</height>
              </rect>
             </property>
             <property name="maximumSize">
              <size>
               <width>450</width>
               <height>16777215</height>
              </size>
             </property>
             <property name="styleSheet">
              <string notr="true">background:#86FF70;
/*margin-top:40px;</string>
             </property>
             <widget class="QLineEdit" name="getName_admin">
              <property name="geometry">
               <rect>
                <x>70</x>
                <y>160</y>
                <width>311</width>
                <height>22</height>
               </rect>
              </property>
              <property name="font">
               <font>
                <pointsize>10</pointsize>
               </font>
              </property>
              <property name="styleSheet">
               <string notr="true">border-radius:0;
border-bottom: 2px solid #383a59;
border-color:#383a59;</string>
              </property>
              <property name="placeholderText">
               <string>Admin</string>
              </property>
             </widget>
             <widget class="QLineEdit" name="get_password">
              <property name="geometry">
               <rect>
                <x>70</x>
                <y>250</y>
                <width>311</width>
                <height>22</height>
               </rect>
              </property>
              <property name="font">
               <font>
                <pointsize>10</pointsize>
               </font>
              </property>
              <property name="styleSheet">
               <string notr="true">border-radius:0;
border-bottom: 2px solid #383a59;
border-color:#383a59;</string>
              </property>
              <property name="echoMode">
               <enum>QLineEdit::Password</enum>
              </property>
              <property name="placeholderText">
               <string>Password</string>
              </property>
             </widget>
             <widget class="QPushButton" name="btn_login_2">
              <property name="geometry">
               <rect>
                <x>160</x>
                <y>350</y>
                <width>121</width>
                <height>41</height>
               </rect>
              </property>
              <property name="font">
               <font>
                <family>Arial</family>
                <pointsize>11</pointsize>
                <weight>75</weight>
                <bold>true</bold>
               </font>
              </property>
              <property name="styleSheet">
               <string notr="true">background-color:#1DB700;
border:none;
</string>
              </property>
              <property name="text">
               <string>Login</string>
              </property>
             </widget>
             <widget class="QPushButton" name="pushButton">
              <property name="geometry">
               <rect>
                <x>140</x>
                <y>10</y>
                <width>171</width>
                <height>101</height>
               </rect>
              </property>
              <property name="styleSheet">
               <string notr="true">image:url(:/newPrefix/pngs/administrator.png)</string>
              </property>
              <property name="text">
               <string/>
              </property>
             </widget>
            </widget>
           </widget>
           <widget class="QWidget" name="tab_2">
            <attribute name="title">
             <string>Tab 2</string>
            </attribute>
            <widget class="QWidget" name="widget_25" native="true">
             <property name="geometry">
              <rect>
               <x>0</x>
               <y>20</y>
               <width>450</width>
               <height>591</height>
              </rect>
             </property>
             <property name="maximumSize">
              <size>
               <width>450</width>
               <height>16777215</height>
              </size>
             </property>
             <property name="styleSheet">
              <string notr="true">background:#86FF70;
/*margin-top:40px;</string>
             </property>
             <widget class="QLineEdit" name="getName_reg">
              <property name="geometry">
               <rect>
                <x>150</x>
                <y>30</y>
                <width>251</width>
                <height>22</height>
               </rect>
              </property>
              <property name="font">
               <font>
                <pointsize>10</pointsize>
               </font>
              </property>
              <property name="styleSheet">
               <string notr="true">border-radius:0;
border-bottom: 2px solid #383a59;
border-color:#383a59;</string>
              </property>
             </widget>
             <widget class="QLabel" name="label_40">
              <property name="geometry">
               <rect>
                <x>40</x>
                <y>30</y>
                <width>71</width>
                <height>21</height>
               </rect>
              </property>
              <property name="font">
               <font>
                <pointsize>10</pointsize>
               </font>
              </property>
              <property name="text">
               <string>Name :</string>
              </property>
             </widget>
             <widget class="QLabel" name="label_41">
              <property name="geometry">
               <rect>
                <x>40</x>
                <y>100</y>
                <width>71</width>
                <height>21</height>
               </rect>
              </property>
              <property name="font">
               <font>
                <pointsize>10</pointsize>
               </font>
              </property>
              <property name="text">
               <string>Sirname :</string>
              </property>
             </widget>
             <widget class="QLineEdit" name="getSirname_reg">
              <property name="geometry">
               <rect>
                <x>150</x>
                <y>100</y>
                <width>251</width>
                <height>22</height>
               </rect>
              </property>
              <property name="font">
               <font>
                <pointsize>10</pointsize>
               </font>
              </property>
              <property name="styleSheet">
               <string notr="true">border-radius:0;
border-bottom: 2px solid #383a59;
border-color:#383a59;</string>
              </property>
             </widget>
             <widget class="QLabel" name="label_42">
              <property name="geometry">
               <rect>
                <x>40</x>
                <y>160</y>
                <width>71</width>
                <height>21</height>
               </rect>
              </property>
              <property name="font">
               <font>
                <pointsize>10</pointsize>
               </font>
              </property>
              <property name="text">
               <string>Mo No  :</string>
              </property>
             </widget>
             <widget class="QLineEdit" name="getMobile_reg">
              <property name="geometry">
               <rect>
                <x>150</x>
                <y>160</y>
                <width>251</width>
                <height>22</height>
               </rect>
              </property>
              <property name="font">
               <font>
                <pointsize>10</pointsize>
               </font>
              </property>
              <property name="styleSheet">
               <string notr="true">border-radius:0;
border-bottom: 2px solid #383a59;
border-color:#383a59;</string>
              </property>
             </widget>
             <widget class="QLineEdit" name="getID_reg">
              <property name="geometry">
               <rect>
                <x>150</x>
                <y>280</y>
                <width>251</width>
                <height>22</height>
               </rect>
              </property>
              <property name="font">
               <font>
                <pointsize>10</pointsize>
               </font>
              </property>
              <property name="styleSheet">
               <string notr="true">border-radius:0;
border-bottom: 2px solid #383a59;
border-color:#383a59;</string>
              </property>
             </widget>
             <widget class="QLabel" name="label_43">
              <property name="geometry">
               <rect>
                <x>40</x>
                <y>340</y>
                <width>91</width>
                <height>21</height>
               </rect>
              </property>
              <property name="font">
               <font>
                <pointsize>10</pointsize>
               </font>
              </property>
              <property name="text">
               <string>Gender:</string>
              </property>
             </widget>
             <widget class="QTextEdit" name="getAdress_reg">
              <property name="geometry">
               <rect>
                <x>150</x>
                <y>390</y>
                <width>251</width>
                <height>87</height>
               </rect>
              </property>
              <property name="font">
               <font>
                <pointsize>10</pointsize>
               </font>
              </property>
              <property name="styleSheet">
               <string notr="true">border: 2px solid #383a59;
border-color:#383a59;</string>
              </property>
              <property name="html">
               <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'MS Shell Dlg 2'; font-size:10pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;br /&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
              </property>
             </widget>
             <widget class="QLabel" name="label_44">
              <property name="geometry">
               <rect>
                <x>40</x>
                <y>390</y>
                <width>91</width>
                <height>21</height>
               </rect>
              </property>
              <property name="font">
               <font>
                <pointsize>10</pointsize>
               </font>
              </property>
              <property name="text">
               <string>Address :</string>
              </property>
             </widget>
             <widget class="QComboBox" name="Gender_reg">
              <property name="geometry">
               <rect>
                <x>150</x>
                <y>340</y>
                <width>101</width>
                <height>22</height>
               </rect>
              </property>
              <property name="font">
               <font>
                <pointsize>10</pointsize>
                <weight>50</weight>
                <bold>false</bold>
               </font>
              </property>
              <item>
               <property name="text">
                <string>Male</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>Female</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>Other</string>
               </property>
              </item>
             </widget>
             <widget class="QComboBox" name="Gender_8">
              <property name="geometry">
               <rect>
                <x>40</x>
                <y>280</y>
                <width>81</width>
                <height>22</height>
               </rect>
              </property>
              <property name="font">
               <font>
                <pointsize>10</pointsize>
                <weight>50</weight>
                <bold>false</bold>
               </font>
              </property>
              <item>
               <property name="text">
                <string>ADHAR</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>PAN</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>VOTER ID</string>
               </property>
              </item>
             </widget>
             <widget class="QLabel" name="label_47">
              <property name="geometry">
               <rect>
                <x>40</x>
                <y>220</y>
                <width>71</width>
                <height>21</height>
               </rect>
              </property>
              <property name="font">
               <font>
                <pointsize>10</pointsize>
               </font>
              </property>
              <property name="text">
               <string>Gmail  :</string>
              </property>
             </widget>
             <widget class="QLineEdit" name="getGmail_reg">
              <property name="geometry">
               <rect>
                <x>150</x>
                <y>220</y>
                <width>251</width>
                <height>22</height>
               </rect>
              </property>
              <property name="font">
               <font>
                <pointsize>10</pointsize>
               </font>
              </property>
              <property name="styleSheet">
               <string notr="true">border-radius:0;
border-bottom: 2px solid #383a59;
border-color:#383a59;</string>
              </property>
             </widget>
             <widget class="QPushButton" name="register_btn_2">
              <property name="geometry">
               <rect>
                <x>160</x>
                <y>530</y>
                <width>121</width>
                <height>41</height>
               </rect>
              </property>
              <property name="font">
               <font>
                <family>Arial</family>
                <pointsize>11</pointsize>
                <weight>75</weight>
                <bold>true</bold>
               </font>
              </property>
              <property name="styleSheet">
               <string notr="true">background-color:#1DB700;
border:none;
</string>
              </property>
              <property name="text">
               <string>Register</string>
              </property>
             </widget>
            </widget>
           </widget>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="my.qrc"/>
 </resources>
 <connections/>
</ui>
